### PR TITLE
Upgrade to v0.15.1

### PR DESCRIPTION
**Pull request by sslavic**
Towards https://github.com/giantswarm/giantswarm/issues/10403

### CODE REVIEW

**Comment by wombat** [comment]
Any update on this?

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Changed
+
+- Upgrade to cert-manager v0.14.2.
```
Comment (sslavic) [inline CHANGELOG.md:12]:
```suggestion
- Upgrade to cert-manager v0.15.1.
```

File: helm/cert-manager-app/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: cert-manager-app
 namespace: kube-system
-appVersion: 0.9.0
+appVersion: 0.14.2
```
Comment (sslavic) [inline helm/cert-manager-app/Chart.yaml:4]:
```suggestion
appVersion: 0.15.1
```

File: helm/cert-manager-app/values.yaml [hunk 3]
```diff
@@ -9,4 +9,5 @@ global:
 
 image:
   name: giantswarm/cert-manager-controller
-  tag: v0.9.0
+  # when updating the tag please make sure to keep appVersion in Chart.yaml in sync
+  tag: v0.14.2
```
Comment (sslavic) [inline helm/cert-manager-app/values.yaml:13]:
```suggestion
  tag: v0.15.1
```

**Comment by sslavic** [comment]
Deprecated in favor of @glitchcrab's work in https://github.com/giantswarm/cert-manager-app/pull/31

**Pull request closed without merging**